03 BusBoard/BusBoard/BusBoard/Models/Arrival.cs

```diff
@@ -0,0 +1,55 @@
+using System.Text.Json.Serialization;
+
+namespace BusBoard.Models;
+
+public class Arrival
+{
+    [JsonPropertyName("lineName")]
+    public required string LineName {
+        get;
+        set; 
+    }
+    
+    [JsonPropertyName("stationName")]
+    public required string StationName {
+        get;
+        set; 
+    }
+
+    [JsonPropertyName("platformName")]
+    public required string PlatformName
+    {
+        get;
+        set;
+    }
+
+    [JsonPropertyName("destinationName")]
+    public required string DestinationName
+    {
+        get;
+        set;
+    }
+
+    [JsonPropertyName("towards")]
+    public required string Towards
+    {
+        get;
+        set;
+    }
+
+    [JsonPropertyName("timeToStation")]
+    public required int TimeToStationInSeconds
+    {
+        get;
+        set;
+    }
+
+    [JsonIgnore]
+    public TimeSpan TimeToStation
+    {
+        get
+        {
+            return TimeSpan.FromSeconds(TimeToStationInSeconds);
+        }
+    }

```

Comment:
This will be recalculated everytime get is called so its not really a field/property. Since TimeSpans are objects they'd all be different instances.
```suggestion
    [JsonIgnore]
    public TimeSpan GetTimeToStation()
    {
        return TimeSpan.FromSeconds(TimeToStationInSeconds);
     }

```